src/libertem/io/dataset/raw.py

```diff
@@ -68,34 +69,39 @@ class RawFileDataSet(DataSet):
     Examples
     --------
 
-    >>> ds = ctx.load("raw", path=path_to_raw, scan_size=(16, 16),
-    ...               dtype="float32", detector_size=(128, 128))
+    >>> ds = ctx.load("raw", path=path_to_raw, nav_shape=(16, 16),
+    ...               dtype="float32", sig_shape=(128, 128), sync_offset=0)
```

Comment:
Maybe can be reordered such that `nav_shape` and `sig_shape` are after one another; maybe put `dtype` to the end?